src/features/Layout/Navigation/components/SupportUsTray.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+
+import CopyToClipboard from 'react-copy-to-clipboard';
+import styled from 'styled-components';
+
+import { Box, Icon, Text } from '@components';
+import { donationAddressMap } from '@config';
+import { useAnalytics } from '@hooks';
+import { ANALYTICS_CATEGORIES } from '@services';
+import { SPACING } from '@theme';
+import { translateRaw } from '@translations';
+import { TTrayItem } from '@types';
+
+import { ExternalLink } from './ExternalLink';
+
+const SText = styled(Text)`
+  text-transform: uppercase;
+`;
+
+export const SupportUsTray = ({ items }: { items: TTrayItem[] }) => {
+  const [displayingMessage, setDisplayMassage] = useState(false);
+  const timeoutId = useRef<ReturnType<typeof setTimeout> | null>(null);
```

Comment:
Prefer `useTimeout` https://github.com/streamich/react-use/blob/master/docs/useTimeout.md
We used to copy selected hooks into `@vendor`.